openwisp_controller/config/static/config/js/device.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
```

Comment:
I think we should rename this file to `management_ip.js`

Reply:
Sure